napari/_qt/widgets/_tests/test_qt_viewer_buttons.py

```diff
@@ -1,5 +1,7 @@
+from unittest.mock import Mock
+
 import pytest
-from qtpy.QtCore import QPoint
+from qtpy.QtCore import QPoint, Qt
 from qtpy.QtWidgets import QApplication
 
 from napari._qt.dialogs.qt_modal import QtPopup
```

Comment:
Did we want to add `mock_app` to the `qt_viewer_buttons` fixture below? @DragaDoncila 

(sorry can't comment that far down)

Reply:
Actually, since `inject` is cached and never gets run, I guess we don't need to. If we decide to make changes re #7219 we could do it then..?

Reply:
@lucyleeow nah we shouldn't need it because we no longer rely on the app in any way